mico-core/src/main/java/io/github/ust/mico/core/model/MicoApplication.java

```diff
@@ -147,6 +151,7 @@
             @ExtensionProperty(name = "description", value = "The information necessary for deploying this application.")
         }
     )})
+    @Deprecated
```

Comment:
Isn't it enough to add the **Deprecated** annotation to the class MicoApplicationDeploymentInfo?